Explorer/Assets/Scripts/ECS/Unity/GLTFContainer/Systems/LoadGltfContainerSystem.cs

```diff
@@ -39,11 +44,22 @@ protected override void Update(float t)
         [None(typeof(GltfContainerComponent))]
         private void StartLoading(in Entity entity, ref PBGltfContainer sdkComponent, ref PartitionComponent partitionComponent)
         {
-            // It's not the best idea to pass Transform directly but we rely on cancellation source to cancel if the entity dies
-            var promise = Promise.Create(World, new GetGltfContainerAssetIntention(sdkComponent.Src, new CancellationTokenSource()), partitionComponent);
-            var component = new GltfContainerComponent(sdkComponent.GetVisibleMeshesCollisionMask(), sdkComponent.GetInvisibleMeshesCollisionMask(), promise);
-            component.State = LoadingState.Loading;
-            World.Add(entity, component);
+            GltfContainerComponent component;
+            if (!sceneData.TryGetHash(sdkComponent.Src, out string hash))
+            {
+                component = GltfContainerComponent.CreateFaulty(sdkComponent.Src);
+                var exception = new ArgumentException($"Asset Bundle {sdkComponent.Src} not found in the content");
+                World.Add(entity, component, new StreamableLoadingResult<GltfContainerAsset>(CreateException(exception)));
```

Comment:
Can you check with `DEBUG_ARCH` if it's not a repetitive adding of ` StreamableLoadingResult`? I have a bit of doubts

Reply:
I ran it, no apparent problem. The componet will be added only if the query of `[None(typeof(GltfContainerComponent))]` is not fullfilled. So, if we have that faulty component added, no repetition should be possible